<ui version="4.0" >
 <class>PatternWizardForm</class>
 <widget class="QWidget" name="PatternWizardForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Параметры образца</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Название испытуемого материала</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lePatternName" />
   </item>
   <item>
    <layout class="QGridLayout" name="_2" >
     <item row="0" column="0" >
      <widget class="QRadioButton" name="rbPatternRectangleSection" >
       <property name="text" >
        <string>Квадратное сечение</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QRadioButton" name="rbPatternRoundSection" >
       <property name="text" >
        <string>Круглое сечение</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Длина образца</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Ширина образца</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Толщина образца</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QDoubleSpinBox" name="dsbPatternLenght" >
       <property name="suffix" >
        <string>мм</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QDoubleSpinBox" name="dsbPatternWidth" >
       <property name="suffix" >
        <string>мм</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QDoubleSpinBox" name="dsbPatternThickness" >
       <property name="suffix" >
        <string>мм</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Описание образца</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="tePatternDescription" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
